tekton/pipeline_run_test.go

```diff
@@ -187,11 +182,7 @@ var _ = Describe("PipelineRun", func() {
 		})
 
 		It("can append the release Name, Namespace, and Workspace to a ReleasePipelineRun object and that these label key names match the correct label format", func() {
-			releasePipelineRun.WithReleaseAndApplicationLabels(
-				release.Name,
-				release.Namespace,
-				release.GetAnnotations()[logicalcluster.AnnotationKey],
-				applicationName)
+			releasePipelineRun.WithReleaseAndApplicationMetadata(release, applicationName)
```

Comment:
should we add an example PaC label/annotation to the release in line 88 and check if they are copied over?

Reply:
Admittedly I wrote that comment before I went through metadata_test.go. I think it could be nice to add here, similarly to how we check the other annotations we expect `WithReleaseAndApplicationMetadata` to add, but it is not required if you don't want to add it

Reply:
In that case I'll mark it resolved, and I'm happy to revisit the topic another time.